tetris.py

```diff
@@ -309,6 +322,14 @@ def move_block_right(self, event):
         self.main_board.user_input_right()
         self.main_board.draw_block()
 
+    def move_block_down(self, event):
+        if self.main_board.moving_down is False:
+            self.main_board.moving_down = True
+        else:
+            self.main_board.moving_down = False
```

Comment:
```suggestion
        if not self.main_board.moving_down:
            return
```

Reply:
Done differently, check out the latest push